src/Lexer.php

```diff
@@ -338,11 +351,12 @@ protected function scanInput(string $regex, string $type)
      * Scans for token with specified keywords.
      *
      * @param string $keywords Keywords (separated by "|")
-     * @param string $type Expected token type
+     *
+     * @phpstan-param TTokenType $type Expected token type
```

Comment:
This method is deprecated and no longer used after #350, I think we can leave the types unchanged.

```suggestion
     * @param string $type Expected token type
```